src/consensus/pbft/libbyz/Status.cpp

```diff
@@ -70,17 +70,11 @@ bool Status::pre_verify()
       // Received message from unknown sender
       LOG_INFO << "Request from unknown pricipal, id:" << id() << std::endl;
 
-      std::string pubk_sig =
-        "aad14ecb5d7ca8caf5ee68d2762721a3d4fdb09b1ae4a699daf74985193b7d42";
-      std::string pubk_enc =
-        "893d4101c5b225c2bdc8633bb322c0ef9861e0c899014536e11196808ffc0d17";
-
       PrincipalInfo info;
```

Comment:
Should this be removed altogether as principals are added through kv hooks?

Reply:
It is still needed in order for backups that are joining to accept the first message from the primary, so that they can then accept the pre-prepares, execute them, and have the kv written to